util/titles.py

```diff
@@ -0,0 +1,62 @@
+from nameparser import HumanName
+from nose.tools import set_trace
+import re
+
+from permanent_work_id import WorkIDCalculator;
+
+
+"""Fallback algorithms for dealing with personal names when VIAF fails us."""
+
+def is_corporate_title(display_name):
```

Comment:
Fair question.  I think, eventually, the pattern for finding a publisher's name present in the title will be different from finding the publisher as the author, but right now they're about same.